src/ampState.js

```diff
@@ -0,0 +1,10 @@
+export default function ampState(data) {
+  if (!data) return '';
+  return Object.entries(data).map(([key, value]) => (
+    `<amp-state id="${key}">
```

Comment:
As far as I'm aware, custom elements that don't start with "x-" are not yet a full standard.